front/lib/swr/conversations.ts

```diff
@@ -365,6 +365,62 @@ export function useAddDeleteConversationTool({
   return { addTool, deleteTool };
 }
 
+export function useVisualizationRevert({
+  workspaceId,
+  conversationId,
+  agentConfigurationId,
+  fileName,
+  fileId,
+}: {
+  workspaceId: string | null;
+  conversationId: string | null;
+  agentConfigurationId: string | null;
+  fileName?: string;
+  fileId: string
+}) {
+  const handleVisualizationRevert = useCallback(
+    async (): Promise<boolean> => {
+      try {
+        const response = await fetch(
+          `/api/w/${workspaceId}/assistant/conversations/${conversationId}/messages`,
+          {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+              content: `Please revert the previous change in ${fileName ? `${fileName} (file ID: ${fileId})` : fileId}.`,
```

Comment:
I only need fileId but it feels weird to see only file ID in user message so added the file name for human friendliness